zio-http/src/main/scala/zhttp/html/Dom.scala

```diff
@@ -16,7 +16,7 @@ sealed trait Dom { self =>
       }
 
       if (elements.isEmpty && attributes.isEmpty) s"$doctype<$name/>"
```

Comment:
While debugging, the problem comes out here.
A quick/hacky way would be adding a `Element.selfClosingTags.contains(name))` bit in the code. But this would break existing cases wherein `<head />` is considered a valid test case.
An updated list of self-closing tags: http://xahlee.info/js/html5_non-closing_tag.html